templates/user-data.sh.tmpl

```diff
@@ -45,7 +25,8 @@ sudo docker run \
   --log-opt max-file=5 \
 %{ endif ~}
   -v $${CONFIG_DIR}:/snowplow/config \
-  -e 'JAVA_OPTS=-Dorg.slf4j.simpleLogger.defaultLogLevel=info' \
+  -e 'JAVA_OPTS=${java_opts}' \
```

Comment:
Have we discussed yet about using the distroless images for open source terraform?  If you did want to switch to distroless, then you would need to change `JAVA_OPTS` to `JDK_JAVA_OPTIONS`.

Reply:
We haven't - this round of changes is focused on getting launch_templates + IMDSv2 in place + auto-scaling rules.  Will leave moving to distroless to another time!